snapshot/snapshot.go

```diff
@@ -331,7 +331,8 @@ func (o *snapshotter) commit(ctx context.Context, isRemote bool, name, key strin
 	}
 
 	if !isRemote { // skip diskusage for remote snapshots for allowing lazy preparation of nodes
-		du, err := fs.DiskUsage(ctx, o.upperPath(id))
+		var du fs.Usage
+		du, err = fs.DiskUsage(ctx, o.upperPath(id))
```

Comment:
Could you add a comment line of notifying that we need to expose `err` to the outer scope?

Reply:
@ktock Will do !

Reply:
@ktock I went a bit further and used a pattern you used in the `createSnapshot` function.  I like what you did there as it has an explicit variable rather than having an err with double duty.

https://github.com/containerd/stargz-snapshotter/blob/9e9e7a507fc58ab7b01ee93401a3ee2461217a40/snapshot/snapshot.go#L517-L524